lib/backdrop.dart

```diff
@@ -298,13 +324,18 @@ class _BackdropScaffoldState extends State<BackdropScaffold>
         child: GestureDetector(
           onTap: () => fling(),
           behavior: HitTestBehavior.opaque,
-          child: SizedBox.expand(
-            child: Container(
-              decoration: BoxDecoration(
-                borderRadius: widget.frontLayerBorderRadius,
-                color: widget.inactiveOverlayColor.withOpacity(0.7),
+          child: Column(
+            children: <Widget>[
+              // if subHeaderAlwaysActive then do not apply inactiveOverlayColor for area with _headerHeight
+              widget.subHeader != null && widget.subHeaderAlwaysActive
+                  ? Container(height: _headerHeight)
```

Comment:
the overlay is not visible because of `subHeaderAlwaysActive` which defaults to `true`.

In "Active back layer" in MDG - it defines two behaviour for subHeader when back layer is revealed - with and without scrim (overlay)

Reply:
should renaming `subHeaderAlwaysActive` -> `subHeaderShowInactiveOverlay` ? I though that would be to long a name.

Reply:
resolving this. If this is not clear or renaming is required the re-open it.

Reply:
Perfect! Sorry, I understood `subHeaderAlwaysActive` wrong then.